server/protocols/jmap/src/main/java/org/apache/james/jmap/model/MailboxFactory.java

```diff
@@ -108,6 +113,9 @@ private Mailbox fromMessageManager(MessageManager messageManager, Optional<List<
         Rights rights = Rights.fromACL(metaData.getACL())
             .removeEntriesFor(Username.forMailboxPath(messageManager.getMailboxPath()));
         Username username = Username.fromSession(mailboxSession);
+        Quotas quotas = new Quotas(ImmutableMap.of(new QuotaId(QuotaRoot.quotaRoot("key")), new Quotas.Quota(ImmutableMap.of(
```

Comment:
we definitely need some factory methods to make that less verbose

Reply:
you don't like new? :) I introduced some factory methods in my last commit.